controls/osquery.rb

```diff
@@ -0,0 +1,17 @@
+# encoding: utf-8
+
+title 'Facebook Osquery'
+
+control 'osquery-1' do
```

Comment:
some would say security is not a requirement in production ;-)
I was planning for optional profile but hadn't the time to do it. will update